sql/create-db.sql

```diff
@@ -1,8 +1,14 @@
-CREATE TABLE `sobytylnik`.`profiles` (
-  `id` BIGINT(19) NOT NULL AUTO_INCREMENT,
-  `name` VARCHAR(45) NOT NULL,
-  `surname` VARCHAR(45) NOT NULL,
-  `age` INT(3) NOT NULL,
-  PRIMARY KEY (`id`))
-ENGINE = InnoDB
-DEFAULT CHARACTER SET = utf8;
\ No newline at end of file
+CREATE TABLE `sobytylnik`.`profiles` (
+  `id` INT NOT NULL AUTO_INCREMENT,
+  `name` VARCHAR(45) NOT NULL,
+  `surname` VARCHAR(45) NULL,
+  `location` VARCHAR(45) NULL,
+  `birthday` VARCHAR(45) NULL,
+  `twitter_page` VARCHAR(45) NULL,
```

Comment:
It's better to use a longer string. For example, VARCHAR(200).
The same for `instagram_page`, `facebook_page`, `telegram_username`